eLibrary.ru.js

```diff
@@ -105,8 +105,8 @@ function scrape (doc) {
 			item.title = doc.title;
 		}
 		item.title = fixCasing(item.title);
-		var title = ZU.xpathText(datablock, "./table[1]");
-		var authors =   ZU.xpath(datablock, './div[1]/table[1]//span/a[contains(@href, "authorid")]');
+		
+		var authors = ZU.xpath(doc, '//table[@width=550]//td[@width=514]/span[@style="white-space: nowrap"]//b');
```

Comment:
The `author_items.asp?authorid` won't work for the [2nd test case](https://elibrary.ru/item.asp?id=9541154) - author name is plain text there, not url.

The `<input type="hidden" name="authors_all" value=...` looks more structured. (Used for "find other papers by same authors" button), but its value is quite strange (see [3rd test case](https://elibrary.ru/item.asp?id=17339044)): 
```
<input type="hidden" name="authors_all" value="Седов Е|Седов Е Н|Седышева Г|Седышева Г А|Серова З|Серова З М|Супрун И|Супрун И И|Ульяновская Е В|Уляновская Е|">
```
 * names are shortened `Седов Е Н` instead of `СЕДОВ ЕВГЕНИЙ НИКОЛАЕВИЧ`
 * names are duplicated `Седов Е|Седов Е Н`
 * names order is incorrect (correct is `СУПРУН ИВАН ИВАНОВИЧ, УЛЬЯНОВСКАЯ ЕЛЕНА ВЛАДИМИРОВНА, СЕДОВ ЕВГЕНИЙ НИКОЛАЕВИЧ, ...`)

thus, this field is hardly useful.

I do agree that xpath, mainly based on `td[@width=514]` doesn't look nice. But, probably, anything else would be just as fragile as this.

I'll try to contact site admins, maybe they'll be so kind to adjust something...

Reply:
Okay, I understand. Thank you for looking this more up in the examples. We can also go here for the path you currently are suggesting and hope that maybe the publisher will change to something more structured website.